lib/create-http-client.js

```diff
@@ -27,6 +29,10 @@ export default function createHttpClient (axios, options) {
   const defaultConfig = {
     insecure: false,
     retryOnError: true,
+    logEmitter: new EventEmitter(),
```

Comment:
isn't `EventEmitter` only available for node, and this library will be bundled with the SDKs for browsers also?